src/common/src/config.rs

```diff
@@ -863,11 +863,28 @@ pub struct FileCacheConfig {
     #[serde(default = "default::file_cache::flush_buffer_threshold_mb")]
     pub flush_buffer_threshold_mb: Option<usize>,
 
+    /// Recovery mode.
```

Comment:
How about adding some descriptions for three RecoverMode ?

Reply:
+1. I wonder whether we can make it a enum instead of a string. example: https://github.com/risingwavelabs/risingwave/blob/e1de1857a6b96c1070d1273cfcebd8eef52a62c3/src/compute/src/lib.rs#L110

Reply:
> How about adding some descriptions for three RecoverMode ?

The descriptions can be found in the foyer doc, let me duplicate it here.

Reply:
> +1. I wonder whether we can make it a enum instead of a string. example: https://github.com/risingwavelabs/risingwave/blob/e1de1857a6b96c1070d1273cfcebd8eef52a62c3/src/compute/src/lib.rs#L110

Agreed. Enum is alreay used in foyer, but no clap/serde support for it. Let me add it.

Reply:
done